index.html

```diff
@@ -254,7 +286,9 @@ <h2 class="title">About Myself</h2>
         </a>
       </li>
     </ul>
+    <div class="cl-btn">
     <a class="btn resume-btn" href="#"> Get my resume</a>
+    </div>
   </div>
   <ul class="knowledges">
```

Comment:
- [ ] Kindly design the knowledge section following the Figma design for the mobile version
- Figma design
<img width="374" alt="Screen Shot 2022-03-25 at 15 53 06" src="https://user-images.githubusercontent.com/28307960/160134355-b57be020-ef80-47fa-9ae1-a570eb115f85.png">
- Your design
<img width="569" alt="Screen Shot 2022-03-25 at 15 52 55" src="https://user-images.githubusercontent.com/28307960/160134405-deab35dd-47c1-4498-ab66-f1d09f7f506d.png">